### DISCUSSION TITLE
Which method should use instead of sendCustomRequest

### DISCUSSION THREAD

**Discussion by yulafezmesi** [WalletConnect/walletconnect-monorepo#2749] [Developer Support] [answered]
hi, currently the app uses `provider.sendCustomRequest`. On WalletConnect v2 which approach should i lead instead?

**Comment by yulafezmesi** [comment, marked as answer]
closing this. i should have add like optionalMethods ['custom_request']

**Reply by benduongquangmobile** [reply]
could you send an example?
